cEP-0009.md

```diff
@@ -0,0 +1,88 @@
+cobot Enhancement and porting
+=============================
+
+|Metadata|                                       |
+|--------|---------------------------------------|
+|cEP     |0009                                   |
+|Version |1.0                                    |
+|Title   |cobot Enhancement and porting          |
+|Authors |Meet Mangukiya <meetmangukiya@coala.io>|
+|Status  |Proposed                               |
+|Type    |Process                                |
+
+Abstract
+--------
+
+This cEP describes the details of the process of porting
+[cobot](https://gitlab.com/coala/cobot) from hubot(coffeescript framework) to
+[errbot](http://errbot.io)(python framework) and about the new features that are
+to be added to cobot as a part of the
```

Comment:
A sample code of a simple hello world command port from hubot to errbot should be mentioned here.